keyboards/foostan/cornelius/readme.md

```diff
@@ -0,0 +1,21 @@
+# cornelius
+
+![cornelius](https://i.imgur.com/kGoUGZgl.jpg)
+
+*Gasket mounted 40% column staggered keyboard manufactured by Yushakobo*
+
+* Keyboard Maintainer: [foostan](https://github.com/foostan/)
+* Hardware Supported: *Cornelius PCBs*
+* Hardware Availability: *http://github.com/foostan*
+
+Make example for this keyboard (after setting up your build environment):
+
+    make foostan/cornelius:via
+
+Flashing example for this keyboard:
+
+    make foostan/cornelius:via:flash
```

Comment:
```suggestion
Gasket mounted 40% column staggered keyboard manufactured by Yushakobo

* Keyboard Maintainer: [foostan](https://github.com/foostan/)
* Hardware Supported: Cornelius PCBs
* Hardware Availability: http://github.com/foostan

Make example for this keyboard (after setting up your build environment):

    make foostan/cornelius:default

Flashing example for this keyboard:

    make foostan/cornelius:default:flash
```

Reply:
@fauxpark Can you give us a reason? I can't understand your suggestion.

Reply:
ah, I see your suggestion.
https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md

It is confusing to include `*` in the template. I thought it was correct to add `*`.

Reply:
The italics are to show it is placeholder text. It's recommended to preview the markdown first so you can see if you've made any formatting errors.